src/StudentsController.ts

```diff
@@ -150,7 +150,14 @@ export class StudentsController {
     }
 
     private async createOnboardingPDF(data: { organizationDisplayName: string; name: string; givenname: string; surname: string; templateReference: string }, pngAsBuffer: Buffer) {
-        const formPdfBytes = await fs.promises.readFile(path.resolve(path.join(this.assetsLocation, "template_onboarding.pdf")));
+        const pathToPdf = path.resolve(path.join(this.assetsLocation, "template_onboarding.pdf"));
+        if (!(await fs.promises.stat(pathToPdf)).isFile()) {
```

Comment:
why don't you call exists?

```suggestion
        if (!fs.existsSync(pathToPdf)) {
```